### PR TITLE
Update Navigation to 2.0.0-rc02

### PR DESCRIPTION
**Pull request by tiembo**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -36,7 +36,8 @@ buildscript {
         ktxVersion = '1.0.1'
         lifecycleVersion = '2.1.0-alpha02'
         materialVersion = '1.0.0'
-        navigationVersion = '1.0.0-rc02'
+        navigationVersion = '2.0.0-rc02'
+        navigationSafeArgsVersion = '1.0.0-rc02'
```
Comment (ianhanniballake) [inline build.gradle]: Navigation and Safe Args should always align in version, so I think it makes sense to collapse these into just a single version number.

File: build.gradle [hunk 2]
```diff
@@ -53,7 +54,7 @@ buildscript {
     dependencies {
         classpath "com.android.tools.build:gradle:$gradleVersion"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-        classpath "android.arch.navigation:navigation-safe-args-gradle-plugin:$navigationVersion"
+        classpath "android.arch.navigation:navigation-safe-args-gradle-plugin:$navigationSafeArgsVersion"
```
Comment (ianhanniballake) [inline build.gradle]: This should be androidx.navigation

File: build.gradle (hunk 1 above)
Comment (tiembo) [reply build.gradle]: Done

File: build.gradle (hunk 2 above)
Comment (tiembo) [reply build.gradle]: Done

**Pull request merged**